### PR TITLE
Thrower Nerf

### PR DESCRIPTION
**Pull request by BlueNothing**
Temporarily removes throwers in a way that's easy to undo if need be.

:cl:
:del: Removes integrated circuit throwers.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Temporarily removes the thrower, as a means of proposing the measure.

Throwers were supposedly intended for use in moving objects between conveyors, into bins, and so on. In practice, they're used as superweapons (chained thrower systems carried in an assembly that kill everything) or as locomotion exploits (circuits can throw themselves, so non-drone circuits can still dart about almost as well, if not better).

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 1]
```diff
@@ -46,6 +46,7 @@ God bless America.
 		/obj/item/reagent_containers/glass,
 		/obj/item/reagent_containers/syringe,
 		/obj/item/reagent_containers/food/condiment,
+		/obj/item/transfer_valve,
```
Comment (Jalleo) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm]: Unstated change

**Comment by Barhandar** [comment]
>or as locomotion exploits (circuits can throw themselves, so non-drone circuits can still dart about almost as well, if not better).

Mental image of a device leapfrogging about is pretty funny, however.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
Don't comment out, remove it proper. If someone wants it back in they'll need to fix it off the webbitor, meaning they can also do a revert on this commit.

**Comment by ExcessiveUseOfCobblestone** [comment]
@BlueNothing yes if someone planned to fix it properly, they wouldn't (shouldn't) use the webeditor which is the only benefit of commenting code out instead of removing it.

**Comment by BlueNothing** [comment]
I didn't know how.

**Comment by ShizCalev** [comment]
#37490

**Pull request closed without merging**